src/App.tsx

```diff
@@ -140,8 +142,13 @@ const wrapProvider = (Component: React.ComponentType<any>, props?: any) => {
 };
 
 const providers = [
+  wrapProvider(PrivyProvider, {
+    appId: import.meta.env.VITE_PRIVY_APP_ID,
```

Comment:
@rosepuppy we have a production Privy app ID ready for you all! since all of the deployers will be sharing this, we can either:
- commit it in-line as a `string` right here in the repo
- add it to `.env.example`
- share the app ID with you all privately to share with deployers

Let us know and we'll make it happen!

Reply:
Lets leave it as an environment variable for now, and to the .env.example. I will be updating it in the docs to recommend the APP ID deployers should use.

Reply:
Sounds good! Will do